pager/src/main/java/com/google/accompanist/pager/Pager.kt

```diff
@@ -362,3 +362,17 @@ private class PagerScopeImpl(
     override val currentPageOffset: Float
         get() = state.currentPageOffset
 }
+
+/**
+ * Calculate the offset for the given [page] from the current scroll position. This is useful
+ * when using the scroll position to apply effects or animations to items.
+ *
+ * The returned offset can positive or negative, depending on whether which direction the [page] is
```

Comment:
Document the range `[-1f .. 1f]` right?

Reply:
Oh wait no, for a 'distant' page this could be outside this range. Are consumers expected to clamp this to be useful for effects?

Is this a smell that `currentPageOffset` doesn't do what consumers expect?

Reply:
I can't make a call on clamping, because we don't know what size the items are (there could be 1, but there could be 5). Each effect is different too, as it could span across all items, or just the center one. 

> Is this a smell that currentPageOffset doesn't do what consumers expect?

Maybe. I don't want to make too many drastic changes now so close to release. This is why I've done this as an ext-fun which can be easily deprecated and removed.